docs/installation_quickstart.rst

```diff
@@ -62,6 +62,10 @@ your desired settings in place of the example environment variables)::
 For more advanced usage of ``ddtrace-run`` refer to the documentation
 :ref:`here<ddtracerun>`.
 
+Once your application is running you can run the command ``ddtrace-run --info``. 
```

Comment:
> Once your application is running

well, this can ("should"?) be used during configuration/onboarding to make sure your configuration is correct before deploying your application?

Reply:
Oftentimes wouldn't a customer start their application with the envars they want though? Or in the case of K8s, they'd deploy their application and it would usually just start as soon as it can when the container starts up right? I guess I don't really see how often the customer would want to try running this command when their application isn't running. It seems like it would be simpler to note that it just doesn't reflect runtime changes?

Reply:
I'm worried about confusing people I guess, since the best use case seems like it would be to usually just start the application with your envars, then run the command. However, I guess we can tell them that you can just run the command with the envars you want to test the connection and settings?

Reply:
Talked with Kyle about how to phrase this better. Making the change.